packages/email/package.js

```diff
@@ -1,6 +1,6 @@
 Package.describe({
   summary: "Send email messages",
-  version: "1.1.15-beta.10"
+  version: "1.1.16-beta.11"
```

Comment:
Thanks for letting me know. There was an HTTP error during the publishing of the `email` package, but I guess it managed to publish a partial build of `email@1.1.15-beta.11`. This will be fixed in the next beta.